docs/proposals/006-scheduler/README.md

```diff
@@ -0,0 +1,150 @@
+# Gateway API Scheduler
+
+Authors: @kfswain, @smarterclayton
+
+## Proposal Status
+ ***Draft***
+
+## Table of Contents
+
+<!-- toc -->
+
+-   [Summary](#summary)
+-   [Goals](#goals)
+-   [Non-Goals](#non-goals)
+-   [Proposal](#proposal)
+    -   [Personas](#personas)
+    -   [Requirements](#requirements)
+    -   [Design](#design)
```

Comment:
do we want to describe in this doc how to configure scheduler during build time in a fork?
or is it too much details for this kind of a doc?

Reply:
I think this is probably too high level - should capture goals and rough components, but let sub proposals cover the doc.